test/functional/p2p_sendheaders.py

```diff
@@ -412,11 +412,13 @@ def test_nonnull_locators(self, test_node, inv_node):
                 test_node.check_last_announcement(inv=[tip], headers=[])
                 if i == 0:
                     # Just get the data -- shouldn't cause headers announcements to resume
```

Comment:
You can remove this comment now that you have a log saying the same thing!

Same for comments lower down.

Reply:
Thanks @jnewbery, appreciate the patience for a newbie, have pushed a new commit remove the comments above the debug prints.